src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -200,7 +324,50 @@ class pt13(Macro):
     def run(self, *args, **kwargs):
         pass
 
+       
+class pt14(Macro):
+    """Macro with list of motors groups, where each motor group is a list of 
+    motors and a float. Repeat parameters may be defined as nested.
+    Usages, ex.:
+    pt14 [[[[mot61] [mot62]] 3]  [[[mot63]] 5] ]
+    pt14 [[mot61 mot62] 3  [mot63] 5]
+    """
+
+    param_def = [
+       ['motor_group_list',
+        [['motor_list', [['motor', Type.Motor, None, 'Motor to move']], None, 'List of motors'],
+         ['float', Type.Float, None, 'Number']],
+        None, 'Motor groups']
+    ]
 
+    def run(self, *args, **kwargs):
+        pass 
+       
+class pt14d(Macro):
+    """Macro with list of motors groups, where each motor group is a list of 
+    motors and a float. Repeat parameters may be defined as nested.
+    Default values can be used.
+    Usages, ex.:
+    pt14d [[[[mot61] [mot62]] 3]  [[[mot63]] 5] ]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.